File: Iniciante/1012.js
let line1 = lines.shift().split(' ');

const pi = 3.14159;

let A = Number(line1.shift());
let B = Number(line1.shift());
let C = Number(line1.shift());

let tri = (A*C)/2;
let cir = pi*(C**2);
let trap = ((A+B)*C)/2;
let qua = B**2;
let ret = A*B;

console.log("TRIANGULO: " + tri.toFixed(3));
console.log("CIRCULO: " + cir.toFixed(3));
console.log("TRAPEZIO: " + trap.toFixed(3));
console.log("QUADRADO: " + qua.toFixed(3));
console.log("RETANGULO: " + ret.toFixed(3));

File: Iniciante/1012.py
linha = input().split()

A = float(linha[0])
B = float(linha[1])
C = float(linha[2])
PI = 3.14159 

aRet = (A*C)/2
aCir = PI*(C**2)
aTra = ((A+B)*C)/2
aQua = B**2
aReta = A*B

print("TRIANGULO: {:.3f}".format(aRet))
print("CIRCULO: {:.3f}".format(aCir))
print("TRAPEZIO: {:.3f}".format(aTra))
print("QUADRADO: {:.3f}".format(aQua))
print("RETANGULO: {:.3f}".format(aReta))
